com.unity.shadergraph/Editor/Data/Graphs/SerializableVirtualTexture.cs

```diff
@@ -4,37 +4,39 @@
 
 namespace UnityEditor.ShaderGraph.Internal
 {
+    // TODO: make TextureType work like this, combine the two enums (needs versioning for old data)
```

Comment:
really wary of shipping anymore "todo" code. can we do this now or make a better note to do it later? todo code almost never gets done

Reply:
I can move it to a feature request in fogbugz instead.   Definitely do not want to make that change in this PR....  :P